pibic-unioeste.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>PIBIC-Unioeste</title>
+    <title-short>PIBIC</title-short>
+    <id>http://www.zotero.org/styles/pibic-unioeste</id>
+    <link href="http://www.zotero.org/styles/pibic-unioeste" rel="self"/>
+    <link href="http://www.zotero.org/styles/pibic-unioeste" rel="template"/>
```

Comment:
The "template" link should point to the starting style. If the comment in the summary is correct, this should be `<link href="http://www.zotero.org/styles/the-holocene" rel="template"/>`